R/crs.R

```diff
@@ -0,0 +1,98 @@
+#' Return the scale factor for a specific parallel in a Lambert Conic Conformal
+#' projection
+#'
+#' The scale factor is the distance distortion.
+#' In this case (a conic conformal projection) it holds in any direction.
+#'
+#' The function applies to normal LCC projections, not the oblique ones.
+#' The applied formulas are for the sphere, hence approximate.
+#'
+#' @return
+#' Numeric vector, same length as \code{par_deg}.
+#'
+#' @family functions to explore properties of coordinate reference systems
+#'
+#' @references
+#' Snyder J.P. (1987). Map Projections - A Working Manual.
+#' U.S. Geological Survey Professional Paper, Nr. 1395. United States
+#' Government Printing Office, Washington, 397 p.
+#'
+#' @param par_deg Numeric vector.
+#' The latitude in decimal degrees of one or more parallels for which to do the
+#' calculation.
+#' @param par1_deg Latitude of the first standard parallel of the LCC projection
+#' in decimal degrees.
+#' @param par2_deg Latitude of the second standard parallel of the LCC
+#' projection in decimal degrees.
+#'
+#' @examples
+#' sf::st_crs(31370)
+#' x <- seq(49.5, 52.5, 0.1)
+#' y <- scalefactor_lcc(x, 51.1666672333333, 49.8333339)
+#' data.frame(latitude = x, scalefactor = y)
+#'
+#' @author
+#' Floris Vanderhaeghe, \url{https://github.com/florisvdh}
+#'
+#' @export
+scalefactor_lcc <- function(par_deg, par1_deg, par2_deg) {
+  par <- par_deg / 180 * pi
```

Comment:
Maybe write this as below. Some people might (incorrectly) think you do `par_deg / (180 * pi)`.
```suggestion
  par <- par_deg * pi / 180
```